app/helpers/application_helper/toolbar/generic_object_definition_center.rb

```diff
@@ -7,11 +7,57 @@ class ApplicationHelper::Toolbar::GenericObjectDefinitionCenter < ApplicationHel
       title,
       :items => [
         button(
+          :generic_object_definition_custom_button_group_new,
+          'pficon pficon-add-circle-o fa-lg',
+          t = N_('Add a new Button Group'),
+          t,
+          :klass     => ApplicationHelper::Button::GenericObjectDefinitionCustomButtonGroupButtonNew,
+          :url_parms => "custom_button_group_new_div"
```

Comment:
You either also need `:send_checked => true` here (assuming the url_parms was to send a list of checked items), or you may want to look into whether this needs to be here at all.

Same for the other entries with url_parms.